debian/clearwater-diags-monitor.init.d

```diff
@@ -101,7 +111,13 @@ do_stop()
         #   1 if daemon was already stopped
         #   2 if daemon could not be stopped
         #   other if a failure occurred
-        start-stop-daemon --stop --quiet --retry=TERM/30/KILL/5 --pidfile $PIDFILE
+        if which start-stop-daemon > /dev/null 2>&1 ; then
+                start-stop-daemon --stop --quiet --retry=TERM/30/KILL/5 --pidfile $PIDFILE
+        else
+                [ ! -f $PIDFILE ] || killproc -p $(cat $PIDFILE)
+                rm -f $PIDFILE
+                pkill -f $DAEMON
```

Comment:
We don't *need* both `killproc` and `pkill`.  Early in development, I saw instances where `killproc` wasn't killing things off (due to bugs) - the pkill is a belt-and-braces approach.

`killproc` without a specified signal does the retry behavior itself (although I was missing the `-d` parameter to enable this, which I've just added).